cms/static/cms/js/modules/cms.modal.js

```diff
@@ -62,54 +88,75 @@
                     closeAndCancel: modal.find('.cms-modal-close, .cms-modal-cancel'),
                     modalButtons: modal.find('.cms-modal-buttons'),
                     modalBody: modal.find('.cms-modal-body'),
-                    iframeHolder: modal.find('.cms-modal-frame'),
+                    frame: modal.find('.cms-modal-frame'),
                     shim: modal.find('.cms-modal-shim')
                 };
             },
 
-            // initial methods
-            _events: function () {
+            /**
+             * Handles all internal events such as maximize/minimize and resizing.
```

Comment:
`Sets up all the event handlers, such as ...` i think would be better, as this method doesn't exactly "handles" these events, but idk